```
Error: /home/runner/work/pandas/pandas/pandas/errors/__init__.py:46:GL03:pandas.errors.PerformanceWarning:Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
Error: /home/runner/work/pandas/pandas/pandas/errors/__init__.py:46:EX02:pandas.errors.PerformanceWarning:Examples do not pass tests:
**********************************************************************
Line 18, in pandas.errors.PerformanceWarning
Failed example:
    df.loc[(1, 'z')]
Expected:
    Traceback (most recent call last):
    PerformanceWarning: indexing past lexsort depth may impact performance.
    df.loc[(1, 'z')])
              jolie
    jim  joe
    1    z        3
Got:
             jolie
    jim joe       
    1   z        3
```

--- a/pandas/errors/__init__.py
+++ b/pandas/errors/__init__.py
@@ -63,7 +63,7 @@ class PerformanceWarning(Warning):
     >>> df.loc[(1, 'z')]
     Traceback (most recent call last):
     PerformanceWarning: indexing past lexsort depth may impact performance.
-    df.loc[(1, 'z')])
+    df.loc[(1, 'z')]
               jolie
     jim  joe
     1    z        3

--- a/pandas/errors/__init__.py
+++ b/pandas/errors/__init__.py
@@ -60,9 +60,8 @@ class PerformanceWarning(Warning):
          x    2
     1    z    3
          y    4
-    >>> df.loc[(1, 'z')]
-    Traceback (most recent call last):
-    PerformanceWarning: indexing past lexsort depth may impact performance.
+    >>> df.loc[(1, 'z')]  # doctest: +SKIP
+    # PerformanceWarning: indexing past lexsort depth may impact performance.
     df.loc[(1, 'z')]
               jolie
     jim  joe

--- a/pandas/errors/__init__.py
+++ b/pandas/errors/__init__.py
@@ -47,7 +47,6 @@ class PerformanceWarning(Warning):
     """
     Warning raised when there is a possible performance impact.
 
-
     Examples
     --------
     >>> df = pd.DataFrame({"jim": [0, 0, 1, 1],

